src/renderer/utils/file-monitor.test.tsx

```diff
@@ -0,0 +1,136 @@
+import fs, { mkdtemp } from 'fs';
+import { sep } from 'path';
+
+import { FileMonitor } from './file-monitor';
+
+let dir: string;
+// The parent directory for the new temporary directory
+mkdtemp(`${__dirname}${sep}`, (err, directory) => {
+  if (err) throw err;
+  dir = directory;
+  fs.rmdirSync(dir);
+});
+
+describe('file-monitor', () => {
+  it('errors should be reported', () => {
+    fs.mkdirSync(__dirname + '/temp');
+    const fileMonitor = new FileMonitor(__dirname + '/temp');
+    const reportedFiles = jest.fn();
+    fileMonitor.getSubject().subscribe({
+      next: (x: unknown) => {
+        reportedFiles(x);
+
+        expect(reportedFiles).toHaveBeenCalledWith(['error']);
```

Comment:
You should pass `reportedFiles()` into `.subscribe()`. And wait for `reportedFiles()` get called.